docs/media-capabilities.md

```diff
@@ -0,0 +1,51 @@
+# Media Capabilities API
+
+It is possible to get the media capabilities and to decide which action to take accordingly.
+
+This API will give you the following information:
+
+- Whether the browser is supporting HEVC or not
+- Whether the browser is supporting DRM or not
+- Which DRM schemas, if any, are supported by the browser (widevine, fairplay, playready)
+- Whether the playback of the media will be power efficient or not
+
+It is optional to pass an object parameter to the API, which will contain the HEVC configuration to test.
+See below a list of the optional properties:
+
+- `width` (number) - Optional width of the video. default is `1920`.
+- `height` (number) - Optional height of the video. default is `1080`.
+- `bitrate` (number) - Optional number of bits used to encode a second of video. default is `1200000`.
+- `framerate` (number) - Optional number of frames used in one second. default is `30`.
+
+The Media Capabilities API is retuning the following object structure:
+
+- `isHEVCSupported` (boolean | string) - Specifies whether HEVC is supported by the browser, or "maybe" if there was a problem with getting the information
+- `isPowerEfficient` (boolean | string) - Specifies power efficiency, or "maybe" if there was a problem with getting the information
+- `isDRMSupported` (boolean | string) - Specifies whether DRM is supported by the browser, or "maybe" if there was a problem with getting the information
```

Comment:
I would recommend use enums:
-1 = maybe
0 = no
1 =  true
or just use a string:
maybe = 'maybe'
no = 'no'
yes = 'yes'

better use one type instead of 2 types, like string or boolean